### PR TITLE
FUnctionality for task 6

### PR DESCRIPTION
**Pull request by DamianMacG**

### CODE REVIEW

File: src/components/CommentList.jsx [hunk 1]
```diff
@@ -0,0 +1,38 @@
+import { useEffect, useState } from "react";
+import { getCommentsByArticleId } from "./utils/utils";
+import CommentCard from "./CommentCard";
+
+const CommentList = ({ article_id }) => {
+  const [comments, setComments] = useState([]);
+  const [isLoading, setIsLoading] = useState(true);
```
Comment (NRMSMITH) [inline src/components/CommentList.jsx]: I would suggest initialising this as false, then setting it to true on the useEffect _before_ the api call!

File: src/components/CommentList.jsx [hunk 2]
```diff
@@ -0,0 +1,38 @@
...
+  }
+
+  return (
+    <div className="comment-list">
+        <h2>Comments:</h2>
+      {comments.map((comment) => (
+        <CommentCard key={comment.comment_id} comment={comment} />
+      ))}
+    </div>
+  );
```
Comment (NRMSMITH) [inline src/components/CommentList.jsx:40]: What happens if there are no comments?

**Review by NRMSMITH — COMMENTED** [review]
This looks great so far! You're storming through :) My advice is on your loading state and to consider what you want if there are no comments on an article :)

**Review by B7Bprog — APPROVED** [review]
Hey Damian, nice refactoring, please merge! 🎉

**Pull request merged**